src/target/riscv/riscv.h

```diff
@@ -152,6 +152,9 @@ struct riscv_info {
 	/* record the tinfo of each trigger */
 	unsigned int trigger_tinfo[RISCV_MAX_TRIGGERS];
 
+	/* record the dpc that triggered it is retired. */
```

Comment:
Please, make sure that comments are accurate and understandable to code readers.

Did you want to say this?

```suggestion
	/* Record if single-step is needed prior to resuming from a software breakpoint or trigger.
	 * Single-step is needed if the instruction that caused the halt was not retired. That is, 
	 * we halted "before" that instruction.
	 */
```

Reply:
addressed, thanks